android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java

```diff
@@ -87,9 +87,11 @@ private void downloadFeed() {
                     feedAdapter.notifyDataSetChanged();
                     handleVisibility();
                 }, throwable -> {
-                    Snackbar.make(swipeRefreshLayout, getActivity()
-                            .getString(R.string.refresh_failed), Snackbar.LENGTH_LONG)
-                            .setAction(R.string.retry_download, view -> refresh()).show();
+                    if(getActivity() != null){
```

Comment:
This is not because the getActivity is null

getActivity will never be null if called after onAttach

The problem is that there is a memory leak. The API call is never disposed and hence it gets triggered on an invalid stale fragment where the activity is not attached. The solution is to add it in a disposable and dispose it at the onStop method or so. This is already being handled in a separate PR about FeedFragment ViewModel